### PR TITLE
The first order of business should be to add a Ruby framework

### PR DESCRIPTION
**Pull request by mskog**
All projects, ever, should begin with picking the correct Ruby framework to base your application on. Since we don't know what this application does yet, I added all of them.

### CODE REVIEW

**Comment by karlingen** [comment]
+1 Approve

**Comment by an0nym0use** [comment]
:+1: Keeps options open

**Comment by bzf** [comment]
:+1:

**Comment by Funfun** [comment]
Haha, +1

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+ruby "2.2.0"
+
+gem "rails"
+gem 'sinatra'
+gem 'cramp'
+gem 'grape'
+gem 'padrino'
+gem 'camping'
+gem 'cuba'
+gem 'lotusrb'
```
Comment (Funfun) [inline Gemfile]: you need "\n" here

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+source 'https://rubygems.org'
+ruby "2.2.0"
```
Comment (karlingen) [inline Gemfile]: we prolly should be using latest version instead: 2.2.1

**Comment by joenas** [comment]
Lul

**Comment by eddwardo** [comment]
+1

**Comment by gulyasm** [comment]
LGTM!
:+1:

**Comment by dfischer** [comment]
:+1:

**Comment by johnbacon** [comment]
+1

**Comment by dd1994** [comment]
:+1

**Comment by jhirbour** [comment]
Don't forget to add Volt: https://github.com/voltrb/volt

**Comment by kyleterry** [comment]
Leave it to the ruby people to come in and steal the thunder only to leave a giant mess behind.

**Pull request merged**